src/components/views/right_panel/EncryptionPanel.js

```diff
@@ -36,7 +36,7 @@ const EncryptionPanel = ({verificationRequest, member, onClose}) => {
         setRequest(verificationRequest);
     }, [verificationRequest]);
 
-    const [phase, setPhase] = useState(request.phase);
+    const [phase, setPhase] = useState(request && request.phase);
```

Comment:
the check below is for undefined so this should be `request ? request.phase : undefined`

Reply:
I believe that's the same as my version...?

```
-> bob && bob.phase
<- undefined
-> var bob = { phase: "foo" }
<- undefined
-> bob && bob.phase
<- "foo"
```

Reply:
depends on the value of bob, I guess in this case it works but is implicit:
![image](https://user-images.githubusercontent.com/2403652/73403545-8e453400-42e7-11ea-956e-d6543f5f78d5.png)

Reply:
Fair enough, a lesson for next time. 😄